OutSystems/outsystems-maps
# ROU-11356: Update Google Maps dependencies

### What was happening
* Some dev dependencies for Google Maps were outdated

### What was done
* Updated the dev dependencies versions on `@types/google.maps` and `browser-sync` 

![image](https://github.com/user-attachments/assets/61fe39da-72b4-4ac4-9d7e-2eabf78b5b91)


### Test Steps
1. Run regression tests on Google Maps
2. Check that everything is working as expected


### Checklist
